site/src/pages/index.astro

```diff
@@ -33,77 +32,199 @@ import MetaLayout from "@/layouts/MetaLayout.astro";
+          <ul><li>The collection links have explicit tabindex attributes, causing them to come before anything else in the page.</li> 
+          <li>The links in the "See More" section are not included in the explicit order, meaning focus tabs to each collection link,
+          then header items, then the "See More" links, then the footer.</li>
+          <li>The explicit tab order is optimized specifically for the widest
+          layout, making it incorrect even within the collections list on narrower viewports.</li></ul>
         </dd>
         <dt>3.2.3: Consistent Navigation</dt>
+        <dd>
+          The links that appear within the "See More" section within the main content are removed from the header navigation where they otherwise typically appear.
+        </dd>
+        <dt>2.4.10: Section Headings </dt>
+        <dd>
+          The page doesn't have an H1. 
+        </dd>  
+      </dl>
+
+<dl slot="wcag3">
+        <dt>Image alternatives</dt>
+        <dd>
+          The alt text on some images is not accurate
+        </dd>
+        <dt>Keyboard only</dt>
+        <dd>
+          The text label on each collection's link only becomes visible on hover (not focus).
+        </dd>
+        <dt>Order of content</dt>
+        <dd>
+          <ul><li>The collection links have explicit tabindex attributes, causing them to come before anything else in the page.</li> 
+          <li>The links in the "See More" section are not included in the explicit order, meaning focus tabs to each collection link,
+          then header items, then the "See More" links, then the footer.</li>
+          <li>The explicit tab order is optimized specifically for the widest
+          layout, making it incorrect even within the collections list on narrower viewports.</li></ul>
+        </dd>
+        <dt>Consistent navigation</dt>
         <dd>
           The links that appear within the "See More" section within the main content
           are removed from the header navigation where they otherwise typically appear.
         </dd>
+        <dt>Section Headings </dt>
+        <dd>
+          The page doesn't have an H1. 
+        </dd> 
       </dl>
     </MetaFailureSection>
 
     <MetaFailureSection href="museum/collections/dishes/" title="Dishes Collection">
+ 
+    <p>This section includes the dishes collection page and related item subpages.</p>
+
+     <p>This section demonstrates content that has very few accessibilty failures and the failures that exist can be worked around or are in less important features. The failures present a mix of those that can be found through automated testing and those that require manual testing.</p>
+
+
       <dl slot="wcag2">
         <dt>1.1.1: Non-text Content</dt>
         <dd>
-          None of the exhibit images have alt text.
+          The exhibit images do not have alt text.
+        </dd>
+        <dt>2.1.1: Keyboard</dt>
+        <dd>
+          The favorites marker is keyboard only.
```

Comment:
Is this supposed to say not accessible via keyboard? (Same goes for wcag3 10 lines below)